lib/templates/test-body-footer.html

```diff
@@ -7,7 +7,7 @@
         try {
           require(file);
         } catch(e) {
```

Comment:
Seems like this param needs to be renamed from `e` to `error`. Either that or change the `console.warn()` below to use `e`.